website/src/pages/edited-collections/edited-collection.page.tsx

```diff
@@ -0,0 +1,63 @@
+import React from "react"
+import { Helmet } from "react-helmet"
+import { Link } from "src/components"
+import * as util from "src/style/utils.css"
+import CWKWLayout from "../cwkw/cwkw-layout"
+import * as css from "../cwkw/cwkw-layout.css"
+import { useDialog } from "./edited-collection-context"
+
+// Renders an edited collection page based on the route parameters.
+const EditedCollectionPage = () => {
+  const dialog = useDialog()
+
+  return (
+    <CWKWLayout>
+      <Helmet>
+        <meta name="robots" content="noindex,nofollow" />
+      </Helmet>
+      <main className={util.paddedCenterColumn}>
+        <article className={dialog.visible ? css.leftMargin : util.fullWidth}>
+          <header>
+            <h1>Cherokees Writing the Keetoowah Way</h1>
+          </header>
+
+          <h3>
+            A digital collection presented by{" "}
```

Comment:
We've talked about this in a few meetings already lol but love the landing page and love the linking back to DAILP!!

Reply:
Tysm!! I'm glad y'all are liking it :)